app_flutter/test/state/flutter_build_test.dart

```diff
@@ -114,7 +116,35 @@ void main() {
       buildState.dispose();
     });
   });
+
+  testWidgets('sign in functions call notify listener',
+      (WidgetTester tester) async {
+    final MockGoogleSignInPlugin mockSignInPlugin = MockGoogleSignInPlugin();
+    when(mockSignInPlugin.onCurrentUserChanged)
+        .thenAnswer((_) => Stream<GoogleSignInAccount>.value(null));
+    when(mockSignInPlugin.signIn()).thenAnswer((_) async => null);
```

Comment:
By default, non mocked methods will return null - which is both awaitable and assignable to future. Does this test pass without these?

Reply:
Thanks! Removed. I started off by trying to test the stream logic of `onCurrentUserChanged` and realized there was a simpler way :)